spec/sessions/start_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+require_relative '../helpers/spec_helper'
+require_relative '../helpers/auth_helper'
+require_relative '../helpers/logs_helper'
+require_relative '../../app/controllers/sessions_controller'
+require_relative '../../app/controllers/records_controller'
+require 'yaml'
+
+data = YAML.load_file('data.yml')
+
+RSpec.describe '[Sessions API]' do
+  include AuthHelper
+  include LoggerHelper
+
+  def app
+    Rack::Builder.new do
+      run ApplicationController
+      use RecordsController
+      use SessionsController
+      use UsersController
+    end.to_app
+  end
+
+  describe '[GET /start]' do
+    before(:each) { clear_cookies }
+
+    context 'when logged in' do
+      before(:each) { log_in('TestUser', 'Test123456!') }
+
+      get '/start'
```

Comment:
I suppose this row should be added to `it` or `before(:each)` block

Reply:
sure, didn't get there yet